ntoskrnl/mm/ARM3/virtual.c

```diff
@@ -5452,11 +5454,17 @@ NtFreeVirtualMemory(IN HANDLE ProcessHandle,
                     // pages to decommit, and then push the VAD's starting address
                     // a bit further down, then decrement the commit charge
                     //
-                    // NOT YET IMPLEMENTED IN ARM3.
-                    //
-                    DPRINT1("Case A not handled\n");
```

Comment:
OK if this else-clause is indeed case A (or any other for that matter), please write in a comment that this is such "case". This allows people to understand what's going on in this whole algorithm and to not loose track.

Reply:
Will do.